pennylane/ops/qubit/matrix_ops.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, *params, wires, do_queue=True):
 
             dim = 2 ** len(wires)
 
-            if U.shape != (dim, dim):
+            if qml.math.shape(U) != (dim, dim):
```

Comment:
Was this a result of the PR?

Reply:
This is something that should have been changed previously but got missed - I noticed it while I was debugging some of the failing tests.